src/opflow/interface/opflowregi.cpp

```diff
@@ -76,10 +76,11 @@ extern PetscErrorCode OPFLOWModelCreate_PBPOLHIOP(OPFLOW);
 
 #if defined(EXAGO_ENABLE_RAJA)
 extern PetscErrorCode OPFLOWModelCreate_PBPOLRAJAHIOP(OPFLOW);
+#endif
+
 #if defined(EXAGO_ENABLE_HIOP_SPARSE)
 extern PetscErrorCode OPFLOWModelCreate_PBPOLRAJAHIOPSPARSE(OPFLOW);
 #endif
-#endif
```

Comment:
This is incorrect. Please revert this change

Reply:
Don't know how those slipped in. I though I chacked that everything built before I checked the code in. It's fixed now.